python-backend/tests/party/resources/test_mgr_appointment_resource.py

```diff
@@ -113,3 +113,28 @@ def test_post_manager_success(test_client, auth_headers):
     assert post_data['party_guid'] == test_manager_data['party_guid']
     assert post_data['mine_guid'] == test_manager_data['mine_guid']
     assert post_resp.status_code == 200
+
+
+def test_post_manager_invalid_date(test_client, auth_headers):
+    # TODO Currently, tests share the same database between test cases in modules. 
+    # This test assumes that it runs directly after test_post_manager_success, so the newest mine manager's start date is today. 
```

Comment:
I haven't tried this yet with pytest, but could we POST twice in the same test to make it independent?

POST a valid date, check that it returns a valid response.. And then POST the invalid date based on the first POSTed date, check that it returns an invalid response?

Just spit-balling solutions here, since I'm not sure on the scope of getting these cleaning up the db properly between test cases.

Reply:
Unfortunately, posting twice wouldn't make the tests independent. The first post is still dependent on the previous test and we wouldn't know beforehand if its valid. 

The real solution is that these tests should all run in isolation, ideally as unit tests with a mock database.